process/block/metablock.go

```diff
@@ -1095,9 +1316,13 @@ func (mp *metaProcessor) CreateBlockHeader(bodyHandler data.BodyHandler, round u
 
 	defer func() {
 		go mp.checkAndRequestIfShardHeadersMissing(round)
+
+		mp.blockSizeThrottler.Add(
```

Comment:
This should be called only when block has been created with success

Reply:
done

Reply:
You should add this also in: if bodyHandler == nil || bodyHandler.IsInterfaceNil() {...

Reply:
throttler is called only if there was a body, and if it was correctly processed.

Reply:
and that check is added in createHeader